ui/infra-compare/InfraCompareTableControls.jsx

```diff
@@ -42,10 +43,14 @@ export default class CompareTableControls extends React.Component {
     );
   };
 
-  filterResult = (platformName, result) => {
+  filterResult = (
+    platformName,
+    result,
+    filterPercent = this.state.filterPercent,
```

Comment:
I think it's a little cleaner to not set default arguments in this situation since this function is only being called once, on line 72. You could even just think about extracting it from `this.state` on line 51 instead if it will only ever be a state value passed as an argument.

Reply:
I have removed the default arguments. But its value will not be this.state.filterPercent each time its called

Reply:
Right, I see that. I think this change looks good though.